src/constants.js

```diff
@@ -193,3 +193,8 @@ export const NFT_GUIDE_URL = 'https://hathor.network/nft-guide';
  * How many token meta we are downloading concurrently
  */
 export const METADATA_CONCURRENT_DOWNLOAD = 10;
+
+/**
+ * Token version
+ */
+export const TOKEN_VERSION = 1;
```

Comment:
I think the parameter name should be more explicit, e.g. `LEDGER_TOKEN_SIGNATURE_VERSION` and the docstring should describe in more detail for what we use it.

Reply:
Changed to `LEDGER_TOKEN_VERSION` since it's used on other commands other than signature.